dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImpl.java

```diff
@@ -137,6 +142,49 @@ public ESContentFactoryImpl() {
         this.languageAPI     =  APILocator.getLanguageAPI();
         this.client          = new ESClient();
         this.indiciesAPI     = APILocator.getIndiciesAPI();
+
+        //Date formats supported by dotCMS in lucene queries
+        LUCENE_DATE_TIME_FORMAT_PATTERN.put("MM/dd/yyyy hh:mm:ssa",
```

Comment:
I think is better idea move this Map and the replaceDateTimeFormatInClause method for its own class